site/mkdocs.yml

```diff
@@ -6,6 +6,7 @@ edit_uri: blob/main/docs/
 
 theme:
   name: material
+  custom_dir: overrides
```

Comment:
The overrides dir is missing in this PR - serve fails with:
```
ERROR   -  Config value 'theme': The path set in custom_dir ('/Users/cht/code/ht/iceberg-rest/iceberg-catalog/site/overrides')
           does not exist.
```

Reply:
Ah yip...removed the only file inside the directory that I was toying with and ergo removed the directory from GitHub. I'll remove this for now. Likely will be added later when Muna does his magic.